yt_dlp/extractor/afreecatv.py

```diff
@@ -229,16 +186,20 @@ def _real_extract(self, url):
             raise ExtractorError(
                 'Video %s has been deleted' % video_id, expected=True)
 
-        station_id = self._search_regex(
-            r'nStationNo\s*=\s*(\d+)', webpage, 'station')
-        bbs_id = self._search_regex(
-            r'nBbsNo\s*=\s*(\d+)', webpage, 'bbs')
         video_id = self._search_regex(
-            r'nTitleNo\s*=\s*(\d+)', webpage, 'title', default=video_id)
+            r'vod.afreecatv.com/player/(\d+)/', webpage, 'title', default=video_id)
 
         partial_view = False
         adult_view = False
         for _ in range(2):
+            video_json = self._download_json(
+                "https://api.m.afreecatv.com/station/video/a/view",
+                video_id, headers={'Referer': url}, data=urlencode_postdata({
+                    'nTitleNo': video_id,
+                    'nApiLevel': 10,
+                }))
+            station_id = video_json['data']['station_no']
+            bbs_id = video_json['data']['bbs_no']
             query = {
                 'nTitleNo': video_id,
                 'nStationNo': station_id,
```

Comment:
single-quote URL, and you could simplify it like this:
```suggestion
            data = self._download_json(
                'https://api.m.afreecatv.com/station/video/a/view',
                video_id, headers={'Referer': url}, data=urlencode_postdata({
                    'nTitleNo': video_id,
                    'nApiLevel': 10,
                }))['data']
            query = {
                'nTitleNo': video_id,
                'nStationNo': data['station_no'],
```
and then on the following line:
```py
                'nBbsNo': data['bbs_no'],
```
(github wouldn't let me include it in the suggestion)